mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  price = price + price / 5;
+  return price;
+}
```

Comment:
Here, you are modifying the argument that's passed to the function. This is usually a bad practice and should be avoided. 

If you would like to learn more, you can look into pure functions.